README.md

```diff
@@ -158,7 +172,9 @@ With reference to the [GitHub Commit status API], the `POST` parameters (`state`
 
 # Source Configuration
 
-## Required keys
+## GitHub commit status only
+
+### Required keys
```

Comment:
Suggestion: `### Required keys (default behaviour)`

Reply:
or maybe not since you already have: `GitHub commit status only`

Reply:
Exactly, Andrea revamped the README adding one subsection per usage type of Cogito (only GitHub, GitHub + chat, only chat). We felt that this was the best way to help the user understand how to use Cogito depending on their goal.